db/migrate/20141218163147_add_share_to_journal_news.journal.rb

```diff
@@ -0,0 +1,24 @@
+# This migration comes from journal (originally 20141204171550)
+class AddShareToJournalNews < ActiveRecord::Migration
+  def up
+    create_join_table :journal_news, :sites do |t|
+       t.primary_key :id
+       t.index [:journal_news_id, :site_id]
+       t.integer :position
+       t.boolean  "front",         default: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.